index.js

```diff
@@ -0,0 +1,103 @@
+
+$(document).ready(()=>{
+  let loadTrips = function loadTrips (){
+    $.get('https://trektravel.herokuapp.com/trips',
+    (response) => {
+      let tripInfo = '';
+      tripInfo+= "<tr><th>ID</th><th>Trip</th><th>Continent</th><th>Weeks</th></tr>";//table headings
+      response.forEach(function(trip) {
+        tripInfo += `<tr data-id=${trip.id}>`
+        for (let attr in trip) {
+          tripInfo += '<td>' + trip[attr] + '</td>'
+        } // for loop
+        tripInfo += '</tr>'
+      })// forEach
+      $('#trips table').html(tripInfo);
+    }) // .get, response
+
+    .fail(function(response){
+      console.log(response);
+      $('#fail').html('<p>Request was unsuccessful</p>')
+    }); // .fail
+  } // loadTrip
+
+
+
+  // Function for AJAX request for a specific trip
+
+  let loadTrip = function loadTrip(id) {
+    $('#trip').html('');
+    $('#ajax-results').html('');
+    $.get(`https://trektravel.herokuapp.com/trips/${id}`,
+      (response) => {
+        console.log(response);
+        let tripInfo = `
+        <div class="row"><h2 class="tripname"> ${response.name} </h2>
+        <p> <b> Continent: </b>${response.continent}</p>
+        <p> <b> About: </b> ${response.about}</p>
+        <p> <b> Category: </b> ${response.category}</p>
+        <p> <b> Weeks: </b>${response.weeks}</p>
+        <p> <b> Cost: </b>${response.cost}</p></div>
+
+        <div  class="row">
+          <p class="reserve"><b> Make a reservation </b></p>
+          <form id="add-trip-form" action = "https://trektravel.herokuapp.com/trips/${id}/reservations">
+          <label for="name">Name:</label>
+          <input type="text" name="name"></input>
+
+          <label for="email">Email:</label>
+          <input type="text" name="email"></input>
+
+          <input type="submit" value="Reserve a spot" class="button"></input>
+          </form>
+        </div> `;
+
+        $('#trip').html(tripInfo);
+      })
+
+      .fail(function(response) {
+        console.log(response);
+        $('#fail').html('<p> Request was unsuccessful </p>')
+      });
+    };
+    $('#trips table').on('click', 'tr', function(){
+      let tripID = $(this).attr('data-id');
+      $('#trip').show();
+      loadTrip(tripID);
+    });
+
+    // Function to submit form to the DB
+    const successCallback = function(response) {
+      console.log("POST request was successful");
+      console.log(response);
```

Comment:
Since this app makes multiple AJAX calls, each of which has a success callback, this function name isn't terribly helpful. The name should tell us what succeeded, what's going to happen next or both. Something like `tripReservationSuccess` might be a better option.